src/main/cc/proxy_client/link_command_inputs.cc

```diff
@@ -0,0 +1,61 @@
+#include "src/main/cc/proxy_client/link_command_inputs.h"
+
+#include <set>
+#include <string>
+#include <sys/stat.h>
+
+namespace remote_client {
+
+using std::set;
+using std::string;
+
+void FindLibrarySearchPaths(int argc, char** argv, set<string>* lib_search_paths) {
+	for (int i = 0; i < argc; ++i) {
+		const string cur(argv[i]);
+		if (cur.length() > 2 && cur[0] == '-' && cur[1] == 'L') {
+			lib_search_paths->insert(cur.substr(2));
+		}
+	}
+}
+
+void FindLinkedLibraries(int argc, char** argv, set<string>* libraries) {
+	for (int i = 0; i < argc; ++i) {
+		const string cur(argv[i]);
+		if (cur.length() > 2 && cur[0] == '-' && cur[1] == 'l') {
+			libraries->insert("lib" + cur.substr(2));
+		}
+	}
+}
+
+bool FileExists(const string& name) {
```

Comment:
This should be refactored into a utils library or something.

Reply:
Done.